config/crd/bases/flinkoperator.k8s.io_flinkclusters.yaml

```diff
@@ -1830,6 +1830,12 @@ spec:
                     - mountPath
                     type: object
                   type: array
+                nodeSelector:
+                  additionalProperties:
+                    type: string
+                  description: 'Selector which must match a node''s labels for the
```

Comment:
node's plz.

Reply:
This is generated, the whole string is inside of a pair of single quote (').